src/resources/js/dropdowns.js

```diff
@@ -359,20 +411,29 @@ var tribe_dropdowns = tribe_dropdowns || {};
 					source: source,
 					search: search,
 					page: page,
-					args: $select.data( 'source-args' )
+					args: $select.data( 'source-args' ),
 				};
 			};
 		}
 
 		// Save data on Dropdown
 		$select.data( 'dropdown', args );
 
-		$container = ( $select.select2( args ) ).select2( 'container' );
+		$container = $select.select2( args );
+
+		// Propagating original input classes to the select2 container.
+		$container.data( 'select2' ).$container.addClass( obj.getSelectClasses( $select ).join( ' ' ) );
 
+		// Propagating original input classes to the select2 container.
+		$container.data( 'select2' ).$container.removeClass( 'hide-before-select2-init' );
+
+		/**
+		 * @todo @bordoni Investigate how and if we should be doing this.
```

Comment:
still working on this?

Reply:
It's ongoing until QA finishes their pass. Hopefully, I can clear that todo before merging.